tools/make-source

```diff
@@ -42,6 +42,6 @@ else
     builddir=.
 fi
 
-make -C $builddir --silent dist-gzip distdir=cockpit-wip 1>&2
+make -j8 -C $builddir --silent dist-gzip distdir=cockpit-wip 1>&2
```

Comment:
Why do we want to hardcode 8 here and not let `-j` pick the default? On some machines with one core this wouldn't necessarily lead to a speedup.